internal/contracts/contracts.go

```diff
@@ -274,4 +275,22 @@ func (mc *JSONContract) Unmarshal() (Contract, error) {
 		mc.StateNonce,
 	}
 	return c, nil
+	
+}
+
+
+
+func GenerateRandomContract() *Contract{
+	b := rand.Read(make([]byte, 32))
+	
+	genSenderPrivKey := ecdsa.GenerateKey(elliptic.P256(), rand.Reader)
+	genrecipPubKeyHash := rand.Read(make([]byte, 32))
+	genVersion := binary.LittleEndian.Uint16(b[:])
+	genSigLen := binary.LittleEndian.Uint8(b[:]);
+	genSignature := rand.Read(make([]byte, genSigLen));
```

Comment:
Let's remove these two `Sig` variables, since we don't need them in the `New` function